bin/zopen

```diff
@@ -45,6 +45,8 @@ Command:
 Options:
   -h, --help, -?    display this help and exit
   -v, --verbose     run in verbose mode
+  --ai              run with tinyllama model via LocalAI by using zopen build, zopen build
```

Comment:
Can you just add --ai to zopen-build? The other commands shouldn't have this option at this point

Reply:
> Can you just add --ai to zopen-build? The other commands shouldn't have this option at this point

I just updated it. Only zopen-build file has option --ai. The other commands won't have this option.